src/libraries/pkg/test/packageTest.targets

```diff
@@ -105,55 +132,27 @@
 
   <Target Name="VerifyNotDependsOnNetStandard"
           DependsOnTargets="ResolveReferences"
-          Condition="'$(_ShortFrameworkIdentifier)' == 'net' AND '$(_TargetFrameworkVersionWithoutV)' &gt;= '4.6.1' AND '$(RuntimeIdentifier)' == '' AND '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
+          Condition="'$(_ShortFrameworkIdentifier)' == 'net' AND '$(_TargetFrameworkVersionWithoutV)' &gt;= '4.6.1' AND '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
     <Error Condition="'$(DependsOnNetStandard)' == 'true'" Text="Package $(TestPackageId) requires netstandard shims when targeting $(TargetFramework)" />
   </Target>
 
   <PropertyGroup>
-    <_testDependsOn>
+    <TestDependsOn Condition="'$(TargetFramework)' != ''">
       LogBeginTest;
       VerifyNotDependsOnNetStandard;
       VerifyReferenceClosure;
       VerifyReferenceTypes;
       VerifyRuntimeClosure;
       VerifyRuntimeTypes;
-    </_testDependsOn>
+    </TestDependsOn>
+    <TestDependsOn Condition="'$(TargetFramework)' == ''">
+      _ComputeTargetFrameworkItems
```

Comment:
nice we are using the cross targeting to iterate over the tfms.

Reply:
I debated just setting `<InnerTargets>Test<InnerTargts>` and this would make build just do the testing.  That would avoid use of private target and item.  Works well for these test-specific projects.  Thoughts on that?

Reply:
yes, i think that would be best. IIRC our build in the current form is already hooked up to Test target.